src/lib/storage/pos_list.hpp

```diff
@@ -62,7 +63,7 @@ struct PosList final : private pmr_vector<RowID> {
   void guarantee_single_chunk() { _references_single_chunk = true; }
 
   // Returns whether the single ChunkID has been given (not necessarily, if it has been met)
-  bool references_single_chunk() const {
+  bool references_single_chunk() const override {
```

Comment:
In the other file, you used final.

Reply:
replaced all occurences of `override` in this file with `final`.